playground/src/markdown/Explainer.md

```diff
@@ -0,0 +1,81 @@
+# What the f*ck? A short explainer
+
+This is a Builder's Playground for the Miden Virtual Machine. You can create any program in Miden Assembly run, debug, prove and verify it. Or just check out our examples. All in the browser.
+
+## What is the Miden VM?
+The Miden VM is a zero-knowledge virtual machine. A regular virtual machine consumes an initial state and a program and produces a final state.
+
+![](https://i.imgur.com/y3yYt2R.png)
+
+The Miden VM works like a regular virtual machine. Only that you can provide secret inputs (Witness) and together with the final state the Miden VM creates a proof for it. A zero-knowledge virtual machines proves that a given set of inputs and program code results in the final state.  
+
+![](https://i.imgur.com/t517366.png)
+
+## Inputs of the Miden VM
+External inputs can be provided to Miden VM in the Playground in two ways:
+
+![](https://i.imgur.com/n0LZpvU.png)
+
+* Public inputs - `stack_init` - can be supplied to the VM by initializing the stack with desired values before a program starts executing. Up to 16 stack items can be initialized in this way.
+* Secret (or nondeterministic) inputs - `advice_tape` - can be supplied to the VM. There is no limit on how much data the advice provider can hold. 
+
+*Check out the comparison example to see how the secret input works*
+
+After a program finishes executing, up to 16 elements can remain on the stack. These elements then become the outputs of the program.
+
+Want to know more? [Here](https://wiki.polygon.technology/docs/miden/intro/overview#inputs-and-outputs).
+
+## Program Code
+![](https://i.imgur.com/xb7Iodq.png)
+
+Our goal is to make Miden VM an easy compilation target for high-level blockchain-centric languages such as Move and Solidity.  However, compilers to help with this have not been developed yet. Thus, for now, the primary way to write programs for Miden VM is to use [Miden assembly](https://wiki.polygon.technology/docs/miden/user_docs/assembly/main).
+
+To get to know the language, check out our examples. It's better than writing in Circuits ...
+
+![](https://i.imgur.com/tyeMjD9.png)
```

Comment:
The right border of the drop down is cut out:

![image](https://user-images.githubusercontent.com/310139/225682553-185d68da-0190-40c9-b646-c6c1c86abae3.png)


Which makes it look weird, it would be nice to make a new shot with the complete dropdown (and removing the editor from the background, that is unrelated to the point here)